docs/content/topics/project-developers/projects.md

```diff
@@ -9,296 +9,267 @@ aliases:
+[Scripting Guide]: /topics/scripting
+
+### Brig init
+
+To quickly bootstrap a new project, Brigade offers the `brig init` command. It
+will generate a new project definition file based on the options provided,
+which can then be used to create the project in Brigade.
+
+For example, to initialize a project named `myproject` with default settings,
+which includes TypeScript as the scripting language and no git configuration,
+run the following:
 
-This same pattern can be used to store a default `brigade.json` configuration file for a project.  The associated prompt is *Upload a default brigade.json config*.
+```console
+$ brig init --id myproject
+```
 
-### Managing Your Projects
+Or, if the alternate scripting language option of JavaScript is preferred, run:
 
-With `brig project create`, your projects are stored in Kubernetes secrets. You
-can save a local copy of that secret using `brigade project create --out=myproject.json`.
+```console
+$ brig init --id myproject --language js
+```
 
-If you have already created the secret, you can fetch it from Kubernetes by running
-`brig project get my/project` where `my/project` is the project name you assigned.
+If the project is git-based, supply the git repository name where the Brigade
+script for this project will reside:
 
-## Creating and Managing a Project (The Old Way)
+```console
+$ brig init --id myproject --git https://github.com/<org>/<repo>.git
+```
 
-Note: Managing Brigade projects via Helm chart is being deprecated in favor of using `brig`.
+A few assets will be created in the directory in which the command is run.
+The bulk of the generated files can be found in the `.brigade` directory,
+including the project definition file (`project.yaml`), a secrets file
+(`secrets.yaml`) and a `NOTES.txt` file with next steps.  Additionally, a
+`.gitignore` file is created to ensure that the secrets file and script
+dependencies are not tracked in version control.
 
-### The Brigade Project Chart
+### The `brig project create` Command
 
-The Brigade Project chart is located in the [brigadecore/charts][charts] source tree at
-`charts/brigade-project`. You can also install it out of the Brigade chart repository.
+With a project definition file in hand, you're now ready to create the project
+with brig:
 
 ```console
-$ helm repo add brigade https://brigadecore.github.io/charts
-$ helm search brigade/brigade-project
-NAME                   	CHART VERSION	APP VERSION	DESCRIPTION
-brigade/brigade-project	1.0.0       	v1.0.0    	Create a Brigade project
+$ brig project create --file project.yaml
 ```
 
-We recommend using the following pattern to create your project:
+This command will submit the project definition to Brigade's API server. After
+validating that the definition adheres to the project schema, Brigade will
+persist the project on the substrate (in the form of a unique namespace) and in
+the backing database.
 
-#### 1. Create a place to store project configs
+### Upgrade a project
```

Comment:
Since the command is `brig project update`, should we stick with "update" terminology instead of "upgrade?"